### PR TITLE
Document multiple arguments.

### PR DESCRIPTION
**Pull request by tasshin**
Document eachnav, and how it makes multiple arguments possible for must, keypath, and nthpath.

### CODE REVIEW

File: List-of-Navigators.md [hunk 1]
```diff
@@ -648,6 +657,15 @@ See also [must](#must)
 [0 :boo]
 ```
 
+`keypath` can now take multiple arguments, for concisely specifying multiple steps. It navigates to each key one after another.
+
+```clojure
+=> (select-one (keypath :a :b) {:a {:b 3}})
```
Comment (nathanmarz) [inline List-of-Navigators.md]: This isn't right.

**Comment by nathanmarz** [comment]
Just noticed one minor issue, I commented inline.

**Comment by tasshin** [comment]
I don't understand, can you clarify?

**Comment by nathanmarz** [comment]
I think you meant to put `(keypath :a)` there like what you did in `must`.

Also, now that I look at it again, I would change those new examples in `keypath` to use strings rather than keywords. There's no reason to use `keypath` with multiple arguments with keywords since you can just do `[:a :b]`

**Comment by tasshin** [comment]
Oh, yeah, I see now what you were saying. Let me give strings a try.

**Comment by tasshin** [comment]
Is this the kind of thing you have in mind for strings?

=> (select-one (keypath "out") {"out" {"in" 3}})
{"in" 3}
=> (select-one (keypath "out" "in") {"out" {"in" 3}})
3

If so, that's in the latest version, feel free to merge. If not, just let me know what you're thinking and I'll fix it :)

**Pull request merged**